contributing_to_docs/doc_guidelines.adoc

```diff
@@ -464,27 +464,35 @@ while the associated `{product-version}` comes from the `name:` fields on any
 
 How these attributes render is dependent on which distro and branch build you
 are viewing. The following table shows the current distros and the
-possible values for `{product-title}` and `{product-version}`.
+possible values for `{product-title}` and `{product-version}`, depending on the branch:
 
 [options="header"]
 |===
 |Distro |`{product-title}` |`{product-version}`
 
 |`openshift-origin`
 |OKD
-|1.2, 1.3, 1.4, 1.5, 3.6, 3.7, 3.9, 3.10, 3.11, Latest
+a|* 3.6, 3.7, 3.9, 3.10, 3.11
+* 4.6, 4.7, 4.8, 4.9, 4.10, 4.11
+* 4 for the `latest/` build from the `main` branch
 
 |`openshift-enterprise`
 |OpenShift Container Platform
-|3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 3.6, 3.7, 3.9, 3.10, 3.11, 4.1, 4.2, 4.3, 4.4, 4.5, 4.6, 4.7, 4.8, 4.9, 4.10, 4.11, 4.12
+a|* 3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 3.6, 3.7, 3.9, 3.10, 3.11
+* 4.1, 4.2, 4.3, 4.4, 4.5, 4.6, 4.7, 4.8, 4.9, 4.10, 4.11, 4.12
 
 |`openshift-dedicated`
 |OpenShift Dedicated
-|Latest
+a|* No value set for the latest `dedicated/` build from the `enterprise-4.11` branch
+* 3 for the `dedicated/3` build from the `enterprise-3.11` branch
+
+|`openshift-rosa`
+|Red Hat OpenShift Service on AWS
+|No values set
```

Comment:
We could probably update the ROSA entry similar to the Dedicated one:
build from the `enterprise-4.11` branch